relayer/chains/mock/message_handlers.go

```diff
@@ -0,0 +1,90 @@
+package mock
+
+import (
+	"fmt"
+
+	chantypes "github.com/cosmos/ibc-go/v3/modules/core/04-channel/types"
+	"github.com/cosmos/relayer/v2/relayer/ibc"
+	"github.com/cosmos/relayer/v2/relayer/provider"
+	"github.com/cosmos/relayer/v2/relayer/provider/cosmos"
+	"go.uber.org/zap"
+)
+
+type MsgHandlerParams struct {
+	mcp           *MockChainProcessor
+	PacketInfo    *chantypes.Packet
+	FoundMessages map[ibc.ChannelKey]map[string]map[uint64]provider.RelayerMessage
+}
+
+var messageHandlers = map[string]func(MsgHandlerParams){
+	ibc.MsgTransfer:        handleMsgTransfer,
+	ibc.MsgRecvPacket:      handleMsgRecvPacket,
+	ibc.MsgAcknowledgement: handleMsgAcknowlegement,
+
+	// TODO handlers for packet timeout, client, channel, and connection messages
+}
+
+func retainMessage(p MsgHandlerParams, channelKey ibc.ChannelKey, message string, sequence uint64, ibcMessage provider.RelayerMessage) {
+	if _, ok := p.FoundMessages[channelKey]; !ok {
+		p.FoundMessages[channelKey] = make(map[string]map[uint64]provider.RelayerMessage)
+	}
+	if _, ok := p.FoundMessages[channelKey][message]; !ok {
+		p.FoundMessages[channelKey][message] = make(map[uint64]provider.RelayerMessage)
+	}
+	p.FoundMessages[channelKey][message][sequence] = ibcMessage
+}
+
+func handleMsgTransfer(p MsgHandlerParams) {
+	channelKey := ibc.ChannelKey{
+		ChannelID:             p.PacketInfo.SourceChannel,
+		PortID:                p.PacketInfo.SourcePort,
+		CounterpartyChannelID: p.PacketInfo.DestinationChannel,
+		CounterpartyPortID:    p.PacketInfo.DestinationPort,
+	}
+	retainMessage(p, channelKey, ibc.MsgTransfer, p.PacketInfo.Sequence, cosmos.NewCosmosMessage(&chantypes.MsgRecvPacket{Packet: *p.PacketInfo}))
+	p.mcp.log.Debug("observed MsgTransfer",
+		zap.String("chainID", p.mcp.chainID),
```

Comment:
We should opt for snake_case vs. camelCase for consistency.
https://github.com/cosmos/relayer/blob/main/docs/logging.md#usage

Reply:
Will do